packages/iris-grid/src/IrisGridProxyModel.ts

```diff
@@ -300,6 +301,10 @@ class IrisGridProxyModel extends IrisGridModel implements PartitionedGridModel {
       !isPartitionedGridModelProvider(this.originalModel) ||
       !this.originalModel.isPartitionRequired
     ) {
+      if (!isPartitionedGridModel(this.originalModel)) {
+        // @ts-expect-error If the original model has an undefined partitionConfig return undefined to make the proxy model also return false in isPartitionedGridModel
```

Comment:
Would it be difficult to change the return type instead of suppressing the error in `partitionConfig` and `isPartitionRequired`?

Reply:
It could be changed, but I think it is correct as it is. A real `PartitionedGridModel` should never return `undefined` for these values, and I think code using a real `PartitionedGridModel` should not need to worry about checking for `undefined`.